### PR TITLE
berithench : new command "agent" is generated

### PR DESCRIPTION
**Pull request by ibiz-kukugi**
1. agent is a berithench cammnd to send multiple transaction while
 running a program
 2. accounts is agent's subcommand to manage accounts of agent

### CODE REVIEW

File: cmd/berithench/agent.go [hunk 1]
```diff
@@ -0,0 +1,584 @@
...
+			txData := make([]byte, 0)
+			if txType == contractCall {
+				txData = common.Hex2Bytes(agent.contracts[(index-int(agent.cfg.TxCount+agent.cfg.StakeCount))%len(agent.contracts)].data)
+			}
+
+			gasLimit := agent.cfg.GasLimit
+			gasPrice := big.NewInt(int64(agent.cfg.GasPrice))
+
+			tx := types.NewTransaction(nonce, to, value, gasLimit, gasPrice, txData, base, target)
+			agent.keystore.Unlock(from, agent.cfg.Password)
```
Comment (mesia777) [inline cmd/berithench/agent.go]: how about unlock addresses before running agent ?
it takes a long time to unlock although done.

File: cmd/berithench/agent.go [hunk 2]
```diff
@@ -0,0 +1,584 @@
...
+	AgentCommand = cli.Command{
+		Name:   "agent",
+		Usage:  "run a agent to transfer token automatically",
+		Action: runAgent,
+		Flags: []cli.Flag{
+			ChainIDFlag,
+			NodesFlag,
+			ConfigFileFlag,
+			KeystoreFlag,
+			PasswordFlag,
```
Comment (mesia777) [inline cmd/berithench/agent.go]: if u want to use same passphrase in test keys, how about adds a passphrase flags?

File: cmd/berithench/agent.go [hunk 3]
```diff
@@ -0,0 +1,584 @@
...
+	}
+
+	client, err := rpc.DialContext(context.Background(), url)
+	if err != nil {
+		agent.errCh <- err
+	}
+
+	var nonceStr string
+
+	err = client.CallContext(context.Background(), &nonceStr, "berith_getTransactionCount", agent.keystore.Accounts()[index].Address, "latest")
```
Comment (mesia777) [inline cmd/berithench/agent.go]: how about use "pending" args instead of "latest" to include pending state ?

File: cmd/berithench/agent.go (hunk 2 above)
Comment (ibiz-kukugi) [reply cmd/berithench/agent.go]: i think keep this flag and change old password flag to passwordDir would be better

Comment (zacscoding) [reply cmd/berithench/agent.go]: > i think keep this flag and change old password flag to passwordDir would be better

ok :)  plz change previous flag :+1:

Comment (mesia777) [reply cmd/berithench/agent.go]: > i think keep this flag and change old password flag to passwordDir would be better

ok :) plz change previous flag 👍

**Pull request merged**